scripts/scr_bomb_world/scr_bomb_world.gml

```diff
@@ -105,17 +136,21 @@ function scr_bomb_world(star_system, planet_number, bombard_target_faction, bomb
 		}else if (strength_reduction==0){
 			txt2+=" no losses from the bombardment.";
 		}
-
+ // Okay, I can see this needs tweaks, just, how can I make it that it checks for 3 conditions, instead of just 2?
+	// Would this work:
+	// if (rel>0 && rel<=20 && (target_strength-strength_reduction)>0){
+		//	txt2+=" minor losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
+	// ?
 		if (rel>0 && rel<=20){
 			txt2+=" minor losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
 		}else if (rel>20 && rel<=40){ 
 	    	txt2+=" moderate losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
 	    }else if (rel>40 && rel<=60){ 
 	    	txt2+=" heavy losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
 	    }else if (rel>60 && (target_strength-strength_reduction)>0){ 
-	    	txt2+=" extreme losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
+	    	txt2+=" devastating losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
 	    }else if ((target_strength-strength_reduction)<=0){ 
-	    	txt2+=" devastating losses from the bombardment.  They have been wiped clean from the planet.";
+	    	txt2+=" total annihilation from the bombardment. They have been wiped clean from the planet.";
 	    }
     
 	    // 135; ?
```

Comment:
```suggestion
		}else if (strength_reduction==0){
			txt2+=" no losses from the bombardment.";
		}
 // Okay, I can see this needs tweaks, just, how can I make it that it checks for 3 conditions, instead of just 2?
	// Would this work:
	// if (rel>0 && rel<=20 && (target_strength-strength_reduction)>0){
		//	txt2+=" minor losses from the bombardment, decreasing "+string(strength_reduction)+" stages.";
	// ?
		if ((target_strength-strength_reduction)<=0){ 
			txt2+=" total annihilation from the bombardment and are wiped clean from the planet.";
		} else {
			var _losses_text = "";
			if (rel>0 && rel<=20) {
				_losses_text = "minor losses";
			} else if (rel>20 && rel<=40) { 
				_losses_text = "moderate losses";
			} else if (rel>40 && rel<=60) { 
				_losses_text = "heavy losses";
			} else if (rel>60 && (target_strength-strength_reduction)>0) { 
				_losses_text = "devastating losses";
			} else {
				_losses_text = "some losses";
			}
			txt2 += $" {_losses_text} from the bombardment, having presence decreased by {strength_reduction}.";
		}
    
	    // 135; ?
```